pkg/lockfile/parse-maven-lock.go

```diff
@@ -60,7 +60,9 @@ func (mld MavenLockDependency) resolvePropertiesValue(lockfile MavenLockFile, fi
 	variablesCount := 0
 
 	interpolationReg := cachedregexp.MustCompile(`\${([^}]+)}`)
+	isMixedReg := cachedregexp.MustCompile(`.+\${[^}]+}|\$[^}]+}.+`)
```

Comment:
[nit] it maybe would be helpful to  add a link to regex101?

Reply:
YES ! Definitely thanks for the reminder :D